src/AirshipBindings.Android.Core/Additions/AirshipWebViewClient.cs

```diff
@@ -6,9 +6,9 @@
 
 using UrbanAirship.Actions;
 
-namespace UrbanAirship.Widget
+namespace UrbanAirship.Webkit
 {
-	public partial class UAWebViewClient
+	public partial class AirshipWebViewClient
```

Comment:
This looks like a change in the API. Does this force us to a major release? Or should we deprecate the old?

Reply:
Not sure, doesn't this just fix the bindings or is it considered a part of the API ?

Reply:
This is a major SDK update and will be versioned as such.